apps/docs/content/guides/platform/pause-restore.mdx

```diff
@@ -0,0 +1,43 @@
+---
+id: 'pause-restore'
```

Comment:
Are we deleting https://supabase.com/docs/guides/platform/migrating-and-upgrading-projects#pause--restore ? Or linking back and forth between the two? Unclear what should go on this page vs the other one

Reply:
I deleted the new page and added the content under a `Time Limits` header of the existing `Caveats` section in `migrating-and-upgrading`